src/components/avatar/Avatar.jsx

```diff
@@ -49,7 +50,8 @@ const Avatar = ({
   imgSrc,
   className,
   link,
-  title,
+  linkLabel,
```

Comment:
I am not sure if having 2 separate props is the best approach here. What is the value if i provide both (alt+linkLabel), can we discuss?

Reply:
I think that this name might be a bit misleading since it doesn't provide visual label to the tree. let's discuss tomorrow approach to aria props